test/integration/cypress/support/page_files/Page.ts

```diff
@@ -5,8 +5,8 @@ export abstract class Page {
     return Cypress.config().baseUrl + this.relativeUrl;
   }
 
-  assertLoaded() {
-    cy.url().should("eq", this.absoluteUrl);
+  assertLoaded(subpath: string = "") {
```

Comment:
This smells like a hack. "subpaths" should be different page objects, possibly with inheritance/composition relations with other page object machinery. It's fine if the page objects in the integration tests don't mirror the page-level components in the front end.